src/modules/src/commander.c

```diff
@@ -174,10 +175,6 @@ static void commanderCrtpCB(CRTPPacket* pk)
   crtpCache.targetVal[!crtpCache.activeSide] = *((struct CommanderCrtpValues*)pk->data);
   crtpCache.activeSide = !crtpCache.activeSide;
   crtpCache.timestamp = xTaskGetTickCount();
-
-  if (crtpCache.targetVal[crtpCache.activeSide].thrust == 0) {

```

Comment:
Understandable. I'll remove this along with the other commander changes I've made.